tests/api/test_monitoring_rest.py

```
@@ -114,3 +116,89 @@ def test_timestamps(app, client):
             }
         }
     }
+
+
+def test_monitoring_forbidden(app, client):
+    """Authenticated user without monitoring role receives 403."""
+    datastore = app.extensions["invenio-accounts"].datastore
+    user = datastore.create_user(email="noperm@test.ch", password="1234", active=True)
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)